frontend/dispatcher.lua

```diff
@@ -123,6 +123,7 @@ local settingsList = {
     skim = {category="none", event="ShowSkimtoDialog", title=_("Skim document"), reader=true},
     back = {category="none", event="Back", title=_("Back"), reader=true},
     previous_location = {category="none", event="GoBackLink", arg=true, title=_("Back to previous location"), reader=true},
+    next_location = {category="none", event="GoForwardLink", arg=true, title=_("Forward to net location"), reader=true},
```

Comment:
(I guess the naming is for consistency with the existing one, but ok, we go back usually from a link - we're not really going forward from/to a link - or there is another event, follow nearest link.)